app/views/works/_standard_form.html.erb

```diff
@@ -185,12 +185,7 @@
                       <%= link_to serial.series.title, serial.series %>
                     </li>
                     <li>
-                      <%= link_to ts('Remove'), serial_work_path(serial),
-                              method: :delete %>
-                    </li>
-                    <li>
-                      <%= link_to ts('Delete'), serial.series,
-                              confirm: ts('Delete series from all works?'),
+                      <%= link_to ts('Remove work from series'), serial_work_path(serial),
```

Comment:
We use title case for our links in most cases, so this should be "Remove Work From Series" (we also use double, not single, quotes). 

Other than that, looks good!